tests/foreman/ui/test_ansible.py

```diff
@@ -602,6 +604,56 @@ def test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors(
             assert f'notice Install the {package_name} package' in notice_log['permission_denied']
             assert f'Installed: rubygem-{package_name}' in notice_log['permission_denied']
 
+    @pytest.mark.no_containers
+    @pytest.mark.rhel_ver_list([settings.content_host.default_rhel_version])
+    def test_positive_debug_log_for_host(
+        self, target_sat, module_org, module_location, module_ak_with_cv
```

Comment:
```suggestion
        self, target_sat, module_org, module_location, module_ak_with_cv, mod_content_hosts
```
I would suggest using `mod_content_hosts` fixture for getting your ContentHosts